src/Type/Definition/IntType.php

```diff
@@ -99,14 +99,11 @@ public function parseValue($value)
     }
 
     /**
-     * @param Node         $valueNode
      * @param mixed[]|null $variables
      *
-     * @return int|null
```

Comment:
can move to return type

Reply:
Can't; violates `LeafScalar`. But I added return type hint to `parseValue`.